rdrf/rdrf/export/model_exporters.py

```diff
@@ -0,0 +1,72 @@
+import os
+from rdrf import models
+from registry.patients import models as patientmodels
+from .catalogue import ModelExporterCatalogue
+from .exporters import ModelExporter
+
+
+# Catalogue with generic exporters
+catalogue = ModelExporterCatalogue()
+
+# Registry specific exporters. Querysets filtered by registry code.
+registry_catalogue = ModelExporterCatalogue()
+
+class RegistryExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(code=self.exporter_context['registry_code'])
+
+
+class RegistryFormExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(registry__code=self.exporter_context['registry_code'])
+
+
+class AdjudicationDefinitionExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(registry__code=self.exporter_context['registry_code'])
+
+
+class SectionExporter(ModelExporter):
+    @staticmethod
+    def get_sections(registry_code):
+        registry = models.Registry.objects.get(code=registry_code)
+        for form in models.RegistryForm.objects.filter(registry=registry):
+            for section_code in form.get_sections():
+                yield models.Section.objects.get(code=section_code)
```

Comment:
@rvl 
* thanks
* absolutely, but for testing it was easier this way. Plus there is some import there as management command already. We will have to decide what to do about the clash.
* Date and time is written out to the META file. I thought that will be enough.
* Yes, good point. Made the change.
* yes I thinks so. @id2359?
* probably yes. @id2359?
* not addressing it now, but if we will I guess it will have to be on import when we detect them, right?